taxonomies/categories.txt

```diff
@@ -22079,12 +22171,23 @@ fr:Chocolats noirs fourrés
 nl:Gevulde pure chocola
 
 <en:Filled chocolates
-<en:Milk chocolates
+<en:Milk chocolates, Milk chocolate filled
```

Comment:
Errors in the categories taxonomy definition:
ERROR - en:filled-milk-chocolates has an undefined parent

Reply:
Merci, corrigé sur le prochain commit